int32 HANDSHAKE_RESULT_ROBOT_BASE_NOT_ALIVE = 0
int32 HANDSHAKE_RESULT_ALREADY_GAINED_CONTROL = 1
int32 HANDSHAKE_RESULT_ALREADY_LOST_CONTROL = 2
int32 HANDSHAKE_RESULT_CONTROL_ACQUIRED = 3
int32 HANDSHAKE_RESULT_CONTROL_REJECTED_ROBOT_BASE_FAULT = 4
int32 HANDSHAKE_RESULT_CONTROL_REJECTED_RC_HALT_TRIGGERED = 5
int32 HANDSHAKE_RESULT_CONTROL_REJECTED_RC_CONTROL_ACTIVE = 6
int32 HANDSHAKE_RESULT_CONTROL_REJECTED_TOKEN_TRANSFER_INTERRUPTED = 7
int32 HANDSHAKE_RESULT_CONTROL_HANDED_OVER = 8
int32 HANDSHAKE_RESULT_CONTROL_REQUEST_TIMEOUT = 9
int32 HANDSHAKE_RESULT_RENOUNCE_REQUEST_TIMEOUT = 10

---
int32 result_code